hs_core/templates/pages/genericresource.html

```diff
@@ -1066,6 +1066,7 @@ <h3 style="color: red" id="file-type-error"></h3>
                 }
 
```

Comment:
Line 1063 also appears to be code to show the save button that can be removed:
$(this).closest("form-control").find("button").show()